components/ws-proxy/pkg/proxy/routes.go

```diff
@@ -188,11 +188,6 @@ func (ir *ideRoutes) HandleSupervisorFrontendRoute(route *mux.Route) {
 }
 
 func (ir *ideRoutes) HandleRoot(route *mux.Route) {
-	if ir.Config.Config.BlobServer == nil {
-		ir.handleRootWithoutBlobserve(route)
-		return
-	}
```

Comment:
[With the change in the `ws-proxy-configmap.yaml` above](https://github.com/gitpod-io/gitpod/pull/3182/files#diff-e94a32e97df84e397163a88899dd32feaf13585c4bafac5eb022a7e526de1ca0R65) it's still possible that the `blobServer` config is missing (when `blobserve` component is disabled). Doesn't that conflict with removing this case here?